.github/workflows/maven.yml

```diff
@@ -0,0 +1,34 @@
+name: Maven
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Test with Java ${{ matrix.jdk }}
+    #runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [ '11', '17', '19' ]
+        dist: [ 'zulu' ]
+        include:
+          - { jdk: '8', dist: 'adopt' } # Zulu doesn't have JDK 8
```

Comment:
zulu stil has v8 -> https://cdn.azul.com/zulu/bin/
is it a big of the related action?